### PR TITLE
111

### PR DESCRIPTION
**Pull request by zvin**

### CODE REVIEW

**Comment by lurch** [comment]
222 :wink:

**Comment by thundron** [comment]
definitely not the best name for a branch :P but it stands for v1.5.111 and the commits make sense

File: lib/gui/app/components/finish/finish.tsx [hunk 1]
```diff
@@ -39,8 +40,19 @@ function restart(goToMain: () => void) {
 	goToMain();
 }
 
+async function getSuccessBannerURL() {
+	return (
+		(await settings.get('successBannerURL')) ??
+		'https://www.balena.io/etcher/success-banner?borderTop=false&darkBackground=true'
```
Comment (lurch) [inline lib/gui/app/components/finish/finish.tsx:46]: I dunno if it's deliberate, or just an accidental inclusion, but if I load that URL in a web-browser and look at the "Network" tab in the WebInspector, I see that there's a request to `https://www.google-analytics.com/analytics.js` ?

Comment (zvin) [reply lib/gui/app/components/finish/finish.tsx:46]: The URL didn't change.
It is not loaded as it is, see https://github.com/balena-io/etcher/blob/12cd8a39c1ec1883474f2027f335100fe3400bf9/lib/gui/app/components/safe-webview/safe-webview.tsx#L91-L94

Comment (lurch) [reply lib/gui/app/components/finish/finish.tsx:46]: Ah cool. Just a quick follow-up reply to confirm that when I try the same thing with
`https://www.balena.io/etcher/success-banner/?borderTop=false&darkBackground=true&optOutAnalytics=true`
then the Network tab in the web-inspector shows that all requests are to `www.balena.io` :+1:

**Comment by nazrhom** [comment]
@balena-ci retest

**Comment by zvin** [comment]
@balena-ci retest

**Review by thundron — APPROVED** [review]

**Pull request merged**